test/lib.js

```diff
@@ -48,14 +50,48 @@ function exec(cmd) {
 }
 
 function forceTick() {
-	return Promise.all([
-		exec(
-			`./bin/validatorWorker.js --single-tick --adapter=dummy --dummyIdentity=awesomeLeader --sentryUrl=http://localhost:8005`
-		),
-		exec(
-			`./bin/validatorWorker.js --single-tick --adapter=dummy --dummyIdentity=awesomeFollower --sentryUrl=http://localhost:8006`
-		)
-	])
+	let leaderTick = `./bin/validatorWorker.js --single-tick --adapter=dummy --dummyIdentity=0xce07cbb7e054514d590a0262c93070d838bfba2e --sentryUrl=http://localhost:8005`
+	let followerTick = `./bin/validatorWorker.js --single-tick --adapter=dummy --dummyIdentity=0xc91763d7f14ac5c5ddfbcd012e0d2a61ab9bded3 --sentryUrl=http://localhost:8006`
+	// using rust validator worker
+	if (process.env.RUST_VALIDATOR_WORKER) {
+		leaderTick = `RUST_BACKTRACE=1 ${
+			process.env.RUST_VALIDATOR_WORKER
+		} -a dummy -i 0xce07cbb7e054514d590a0262c93070d838bfba2e -u http://localhost:8005 -t`
+		followerTick = `RUST_BACKTRACE=1 ${
+			process.env.RUST_VALIDATOR_WORKER
+		} -a dummy -i 0xc91763d7f14ac5c5ddfbcd012e0d2a61ab9bded3 -u http://localhost:8006 -t`
+	}
+
+	return Promise.all([exec(leaderTick), exec(followerTick)])
+}
+
+function nonce() {
```

Comment:
is there a more elegant way to do this?

Reply:
I made the change to use current timestamp as `nonce`